ComponentFramework3.0/ComponentFramework/Scene0.cpp

```diff
@@ -119,23 +174,28 @@ void Scene0::processInput(const SDL_Event &SDLEvent, float deltaTime)
 		sceneCamera->ProcessKeyboard(CAMERA::LEFT, deltaTime);
 		printf("Move left");
 		break;
-
 	case SDLK_d:
 		sceneCamera->ProcessKeyboard(CAMERA::RIGHT, deltaTime);
 		printf("Move RIGHT");
-			break;
+		break;
+	case SDLK_l:
+		placeObjects("chair.obj");
```

Comment:
hard coded chair object on key press is probably not something you would commit to master, however i understand this is a test branch.